stock_voucher/models/stock_picking.py

```diff
@@ -135,14 +128,13 @@ def product_onchange(self):
     def do_print_voucher(self):
         '''This function prints the voucher'''
         report = self.env['ir.actions.report'].search(
-            [('report_name', '=', 'stock_voucher.report')], limit=1).report_action(self)
-        # funcionalidad depreciada
-        # if self._context.get('keep_wizard_open', False):
-        #     report['type'] = 'ir.actions.report_dont_close_xml'
+            [('report_name', '=', 'stock.report_picking')],
+            limit=1).report_action(self)
         return report
 
-    @api.one
+    @api.multi
     def assign_numbers(self, estimated_number_of_pages, book):
+        self.ensure_one()
         for page in range(estimated_number_of_pages):
             name = book.sequence_id.next_by_id()
             self.env['stock.picking.voucher'].create({
```

Comment:
`self.env['stock.picking.voucher']`: define before the for loop in a variable and the used inside the for loop.